components/board/pi/common/errors.go

```diff
@@ -0,0 +1,151 @@
+package picommon
+
+// PiGPIOErrorMap maps the error codes to the human readable error names. This can be found at the pigpio C interface.
+var PiGPIOErrorMap = map[int]string{
```

Comment:
@randhid Do you think this is the correct location for this file?

Reply:
yep.